### ISSUE TITLE
S3 bucket name change

### ISSUE THREAD

**Issue by subhankarb** [frictionlessdata/dpr-api#154]
right now the bucket name of s3 is bits.{{domain_name}}. But the problem is that
```
When using virtual hosted–style buckets with SSL, the SSL 
wildcard certificate only matches
buckets that do not contain periods. To work around this, 
use HTTP or write your own certificate
verification logic. We recommend that you do not use periods (".") 
in bucket names.
```
So we have to change the naming convention of the bucket.

**Comment by rufuspollock** [comment]
@subhankarb as i understand it the fix to this is to use cloudfront e.g. https://www.h3xed.com/web-development/using-https-with-amazon-s3-and-your-domain

What is the cost of doing this?

The two options going forward are:

- Use buckets which do not have "." in them - which breaks having nice subdomains e.g. bits.{our-main-domain}.com
- Use cloudfront - if that works.

The latter is def preferable. Can you check this out.

**Comment by rufuspollock** [comment]
@subhankarb what is your plan here? Also, let's make sure if we do use cloudfront that it is automated.

**Comment by subhankarb** [comment]
@rufuspollock sure

**Comment by subhankarb** [comment]
Manual fix for right now will be addressed in #166
